qmri_mpm/sub-01/fmap/sub-01_echo-1_flip-01_TB1EPI.json

```diff
@@ -0,0 +1,43 @@
+{
+  "Manufacturer": "SIEMENS ",
+  "ManufacturersModelName": "Prisma_fit",
+  "DeviceSerialNumber": "167025",
+  "StationName": "MRC35437",
+  "MagneticFieldStrength": 3,
+  "ScanningSequence": "RM",
+  "SequenceName": "seste1d3d2",
+  "PulseSequenceDetails": "mfc_seste_b1map_v1e ",
+  "RepetitionTimeExcitation": 0.5,
+  "EchoTime": 0.03906,
+  "FlipAngle": 65,
+  "MTState": false,
+  "NumberShots": 11,
+  "TotalReadoutTime": 6720.006240000001,
```

Comment:
We are getting a warning for the `TotalReadouttime` fields in the `qmri_mpm` datasets:

Seems valid:

![image](https://user-images.githubusercontent.com/9084751/107845823-d04ff380-6dde-11eb-97ca-948b49339027.png)